primitives/src/transaction.rs

```diff
@@ -11,8 +11,9 @@
 //! This module provides the structures and functions needed to support transactions.
 
 use core::fmt;
-
```

Comment:
There are a few of these where you are just removing a line for no apparent reason which makes the PR more messy.  I'd avoid this for the reviewers.